openadapt/config.py

```diff
@@ -38,8 +56,73 @@ def getenv_fallback(var_name):
 ROOT_DIRPATH = pathlib.Path(__file__).parent.parent.resolve()
 DB_FPATH = ROOT_DIRPATH / DB_FNAME
 DB_URL = f"sqlite:///{DB_FPATH}"
+DIRNAME_PERFORMANCE_PLOTS = "performance"
 
 if multiprocessing.current_process().name == "MainProcess":
     for key, val in locals().items():
         if not key.startswith("_") and key.isupper():
             logger.info(f"{key}={val}")
+
+
+# ACTION EVENT CONFIGURATIONS
+TEXT_SEP = "-"
+TEXT_NAME_PREFIX = "<"
+TEXT_NAME_SUFFIX = ">"
+
+
+# SCRUBBING CONFIGURATIONS
+SCRUB_CHAR = "*"
+SCRUB_LANGUAGE = "en"
+SCRUB_CONFIG_TRF = {
+    "nlp_engine_name": "spacy",
+    "models": [{"lang_code": "en", "model_name": "en_core_web_trf"}],
+}
+SCRUB_PROVIDER_TRF = NlpEngineProvider(nlp_configuration=SCRUB_CONFIG_TRF)
+NLP_ENGINE_TRF = SCRUB_PROVIDER_TRF.create_engine()
+ANALYZER_TRF = AnalyzerEngine(
+    nlp_engine=NLP_ENGINE_TRF, supported_languages=["en"]
+)
+ANONYMIZER = AnonymizerEngine()
+IMAGE_REDACTOR = ImageRedactorEngine(ImageAnalyzerEngine(ANALYZER_TRF))
```

Comment:
Please define the following in `scrub.py`:
```
SCRUB_PROVIDER_TRF
NLP_ENGINE_TRF
ANALYZER_TRF
ANONYMIZER
IMAGE_REDACTOR
```

Reply:
Done!